docs/development.md

```diff
@@ -1,107 +1,159 @@
 # Development
 
-This document should entail all you need to develop this operator locally. 
+- [Development](#development)
+  - [Development Environment Setup](#development-environment-setup)
+    - [golang](#golang)
+    - [operator-sdk](#operator-sdk)
+  - [Dependencies](#dependencies)
+  - [Linting](#linting)
+  - [Testing](#testing)
+  - [Building](#building)
+  - [Build using boilerplate container](#build-using-boilerplate-container)
+  - [How to run](#how-to-run)
+    - [Locally](#locally)
+    - [Remotely](#remotely)
+
+This document should entail all you need to develop this operator locally.
 
 ## Development Environment Setup
 
 ### golang
 
 A recent Go distribution (>=1.13) with enabled Go modules.
 
-```
+```shell
 $ go version
-go version go1.13.15 linux/amd64
-$
+go version go1.16.3 linux/amd64
```

Comment:
is this a requirement of new sdk version?

Reply:
negative, this is the version that happens to be on my machine and I tested the steps with. Suggestion at the top is still the same- `A recent Go distribution (>=1.13) with enabled Go modules.`